src/Parallel/ConstGlobalCache.hpp

```diff
@@ -234,7 +234,20 @@ struct ConstGlobalCacheImpl : ConstGlobalCache, db::SimpleTag {
 /// Tag used to retrieve data from the `Parallel::ConstGlobalCache`. This is the
 /// recommended way for compute tags to retrieve data out of the global cache.
 template <class CacheTag>
-struct FromConstGlobalCache : CacheTag, db::ComputeTag {
+struct FromConstGlobalCache : CacheTag, db::SimpleTag {
+  static std::string name() noexcept {
+    return "FromConstGlobalCache(" + pretty_type::short_name<CacheTag>() + ")";
+  }
+  using type = typename CacheTag::type;
```

Comment:
This still doesn't work...

Reply:
see my comment above please

Reply:
https://github.com/sxs-collaboration/spectre/pull/1583#issuecomment-505876162